scalableminds/webknossos
# Manage User Scripts in wK

Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Adds support for managing a collection of user scripts to wK 
- Scripts need to be hosted as Github Gists
- create, edit, delete scripts from the UI
- Script can be added to tasks and will be automatically executed once you start a task tracing 

Steps to test:
- Create / edit / delete scripts in the UI `/scripts`
- Create / edit a task and add a script to it
- Retrieve that task and make sure the script is executed

Link to Merge Mode Script https://gist.github.com/kevinbbb87/1e1018672981ff6eda0c30588d973074

Issues:
- fixes #1168 

------
